hotel_housekeeping/models/hotel_housekeeping.py

```diff
@@ -0,0 +1,234 @@
+# See LICENSE file for full copyright and licensing details.
+
+import time
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+from odoo import api, fields, models, _
+from odoo.osv import expression
+from odoo.exceptions import ValidationError
+
+DONE_STATE = {'done': [('readonly', True)]}
```

Comment:
this thingy is defined outside the class, thus it couldn't be altered via inheritants